manifests/h/Huawei/AppGallery/13.4.1.300/Huawei.AppGallery.installer.yaml

```
@@ -0,0 +1,18 @@
+# Created with YamlCreate.ps1 v2.2.13 $debug=NVS1.CRLF.7-4-0.Win32NT
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.5.0.schema.json
+
+PackageIdentifier: Huawei.AppGallery
+PackageVersion: 13.4.1.300
+InstallerType: nullsoft
+InstallerSwitches:
+  Upgrade: --updated
+Scope: machine
+UpgradeBehavior: install
+ReleaseDate: 2023-11-02
+Installers:
+- Architecture: x64
+  InstallerUrl: https://store-drcn.hispace.dbankcloud.cn/dl/appdl/application/apk/98/98121a6b62c745e6bf4b08402d31f803/PC665941230b504006a0d4115e92f79f75.2311021425.exe?maple=0&trackId=0&distOpEntity=HWSW
```

```suggestion
  InstallerUrl: https://appdl-11-drcn.dbankcdn.com/dl/appdl/application/apk/98/98121a6b62c745e6bf4b08402d31f803/PC665941230b504006a0d4115e92f79f75.2311021425.exe?maple=0&trackId=0&distOpEntity=HWSW
```
Hi @SpecterShell,

I got a slightly different InstallerUrl. Thoughts?

---

The first URL the link on the homepage redirects to is similar to this one.